system_test/aest_peers_SUITE.erl

```diff
@@ -187,6 +191,22 @@ test_inbound_limitation(Cfg) ->
 setup(Nodes, Config, Cfg) ->
     setup_nodes([maps:put(config, Config, N) || N <- Nodes], Cfg).
 
+wait_for_internal_api(Nodes) ->
+    wait_for_internal_api(Nodes, Nodes).
+
+wait_for_internal_api(Nodes, []) -> ok;
+wait_for_internal_api(AllNodes, [Node | Rest]) ->
+    Res = aehttp_client:request('GetPeers', #{}, [
+                {int_http, aest_nodes_mgr:get_service_address(Node, int_http)},
+                {ct_log, true}
+    ]),
+    case Res of
+        {ok, _, _} -> wait_for_internal_api(AllNodes, Rest);
+        _Error ->
+            timer:sleep(200),
+            wait_for_internal_api(AllNodes, AllNodes)
+    end.
+
 get_peers(Node) ->
     {ok, 200, Peers} = aehttp_client:request('GetPeers', #{}, [
```

Comment:
I see [it was](https://circleci.com/gh/aeternity/epoch/21991):
```
%%% aest_peers_SUITE ==> {{badmatch,{error,socket_closed_remotely}},
 [{aest_peers_SUITE,get_peers,1,
                    [{file,"/home/circleci/epoch/_build/system_test+test/extras/system_test/aest_peers_SUITE.erl"},
                     {line,191}]},
  {aest_peers_SUITE,test_inbound_limitation,1,
                    [{file,"/home/circleci/epoch/_build/system_test+test/extras/system_test/aest_peers_SUITE.erl"},
                     {line,134}]},
  {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1546}]},
  {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1062}]},
  {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,994}]}]}
```

That was this line https://github.com/aeternity/epoch/blob/efe39cc79e7da76c6525e63b59b51cb976c028e8/system_test/aest_peers_SUITE.erl#L191